pstress/mysqld_options_ms_80.txt

```diff
@@ -1619,14 +2000,50 @@
 --log-queries-not-using-indexes=0
 --log-raw=1
 --log-raw=0
+--log-replica-updates=1
+--log-replica-updates=0
 --log-short-format=1
 --log-short-format=0
 --log-slave-updates=1
 --log-slave-updates=0
 --log-slow-admin-statements=1
 --log-slow-admin-statements=0
+--log-slow-extra=1
+--log-slow-extra=0
+--log-slow-filter=qc_miss
+--log-slow-filter=full_scan
+--log-slow-filter=full_join
+--log-slow-filter=tmp_table
+--log-slow-filter=tmp_table_on_disk
+--log-slow-filter=filesort
+--log-slow-filter=filesort_on_disk
+--log-slow-rate-limit=1
+--log-slow-rate-limit=0
+--log-slow-rate-limit=1
+--log-slow-rate-limit=2
+--log-slow-rate-limit=12
+--log-slow-rate-limit=24
+--log-slow-rate-limit=254
+--log-slow-rate-limit=1023
+--log-slow-rate-limit=2047
+--log-slow-rate-limit=-1125899906842624
+--log-slow-rate-limit=1125899906842624
+--log-slow-rate-type=session
+--log-slow-rate-type=query
+--log-slow-replica-statements=1
+--log-slow-replica-statements=0
 --log-slow-slave-statements=1
 --log-slow-slave-statements=0
+--log-slow-sp-statements=1
+--log-slow-sp-statements=0
+--log-slow-verbosity=microtime
```

Comment:
Add log-slow-verbosity to exclusion list

Reply:
Done